### PR TITLE
New UI - Edit Profile Page

### PR DESCRIPTION
**Pull request by yash15601**
### Screenshots of the changes (If any) -
Desktop view:
Screenshot (72)

Mobile view:
image

### CODE REVIEW

**Comment by yash15601** [comment]
@nitin10s @aayush-05  Review this Edit profile page ui

**Comment by aayush-05** [comment]
Also, in the line below Edit Profile, the asterisk should come before "are". And try using class "btn-about" for save button and maybe increase the label text size.

**Comment by yash15601** [comment]
Will do this today 😇

**Comment by yash15601** [comment]
@nitin10s  @aayush-05  Review please

**Comment by yash15601** [comment]
@nitin10s  its almost over i fixed that you can see screenshot

Screenshot (70)

**Comment by yash15601** [comment]
@nitin10s  Review

**Comment by nitin10s** [comment]
@yash15601 Please fix the code climate issues.

**Comment by yash15601** [comment]
@nitin10s  review  please all the issues are basic issues not to worry about i have fixed some of them

**Comment by nitin10s** [comment]
FireShot Capture 163 - CircuitVerse - Digital Circuit Simulator - 127 0 0 1
The inputs are of different width

**Comment by yash15601** [comment]
@nitin10s  
Screenshot (67)
its working fine you can see this

**Comment by nitin10s** [comment]
Please fix the codeclimate issues.

**Comment by yash15601** [comment]
i will fix the code climate and also squash the commit

**Comment by aayush-05** [comment]
-Fix codeclimate, accesslint issues and mark the ones as resolved which you have fixed
-Make the save button a little bigger
-Don't remove any code, add your additional classes if they are conflicting
image
-This haven't been fixed (try using the asterisk code (&#42;) instead)
image
@yash15601

**Comment by yash15601** [comment]
@aayush-05  work in progress

File: app/assets/stylesheets/layout.scss [hunk 1]
```diff
@@ -1,4 +1,72 @@
...
+#profile input[type="text"], select {
+  height: 50px !important;
+  width: 100%;
+}
+
+.btn-secondary:hover {
+  border: 2px solid $green;
+}
+
+.btn-secondary {
```
Comment (tachyons) [inline app/assets/stylesheets/layout.scss]: Do not override standard bootstrap classes

**Comment by tachyons** [comment]
Can you close this and open new PR once linting issues are resolved, difficult to track here as too many linting error messages

**Comment by yash15601** [comment]
@tachyons sure will open new one

**Pull request closed without merging**